assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -199,7 +209,9 @@ function SizePositionPanel({
           suffix={__('Rotate', 'web-stories')}
           symbol={_x('°', 'Degrees, 0 - 360. ', 'web-stories')}
           value={rotationAngle}
-          onChange={(value) => pushUpdate({ rotationAngle: value })}
+          onChange={(value) =>
+            pushUpdate({ rotationAngle: formatRotationAngleField(value) })
```

Comment:
`value === MUTLIPLE_VALUE` and `value === ''` are valid values here. I recommend something like `typeof value === 'number' ? value % 360 : value`